.codecov.yml

```diff
@@ -9,3 +9,9 @@ coverage:
       default:
         target: 20% # Need to bump this number https://docs.codecov.com/docs/commit-status#patch-status
         threshold: 2%
+
+ignore:
+  - "tests"
+  - "src/data/cassandra/"
+  - "src/data/CassandraBackend.hpp"
+  - "src/data/BackendFactory.*"
```

Comment:
The thing is that if we run `codecov/codecov-action` action where build files exist, it reruns the gcovr, but without exclude list, which makes the coverage value different.